postcss.config.js

```diff
@@ -1,6 +1,19 @@
+const path = require('path');
+const { execSync } = require("child_process");
+
+const themeRoot = execSync(`bundle show bullet_train-themes-${process.env.THEME} 2> /dev/null`).toString().trim()
+const themeStylesheets = path.resolve(themeRoot, 'app/assets/stylesheets/');
+
 module.exports = {
   plugins: [
-    require('postcss-import'),
+    require('postcss-import')({
+      resolve: (id, basedir, importOptions) => {
+        if (id.startsWith('$THEMESTYLESHEETS')) {
+          id = id.replace('$THEMESTYLESHEETS', themeStylesheets);
+        }
+        return id;
+      }
+    }),
```

Comment:
This is really cool! Everything in https://github.com/bullet-train-co/bullet_train/issues/567 seems pretty in depth, so I'm glad you've found a solution for local overrides.

By the way, I believe we'll need to add the same logic to `postcss.mailer.config.js` and append the `THEME` environment variable to the build script in `package.json` like so:

```js
"light:build:mailer:css": "THEME=\"light\" ..."
```

Here's the error I was getting for reference:
```
light-mailer-css | Error: Failed to find '$THEMESTYLESHEETS/light/application.css'
light-mailer-css |   in [
light-mailer-css |     /home/gazayas/work/bt/bullet_train/local/bullet_train-core/bullet_train-themes-light/app/assets/stylesheets
light-mailer-css |   ]
light-mailer-css |     at /home/gazayas/work/bt/bullet_train/node_modules/postcss-import/lib/resolve-id.js:35:13
light-mailer-css |     at async LazyResult.runAsync (/home/gazayas/work/bt/bullet_train/node_modules/postcss/lib/lazy-result.js:261:11)
light-mailer-css |     at async Object.watch (/home/gazayas/work/bt/bullet_train/node_modules/tailwindcss/lib/cli/build/plugin.js:375:13)
light-mailer-css |     at async build (/home/gazayas/work/bt/bullet_train/node_modules/tailwindcss/lib/cli/build/index.js:47:9)
```

Reply:
I'm seeing the same error about the mailer styles.